paz/processors/keypoints.py

```diff
@@ -211,10 +226,79 @@ class ComputeOrientationVector(Processor):
     # Returns
         orientation: Array. Orientation of keypoint links
     """
+
     def __init__(self, parents):
         super(ComputeOrientationVector, self).__init__()
         self.parents = parents
 
     def call(self, keypoints):
         orientation = compute_orientation_vector(keypoints, self.parents)
         return orientation
+
+
+def relative_to_absolute(relative_position, width, height):
```

Comment:
could go into `paz.backend.keypoints`. This works for one keypoint but you could easily make it such that it works for many i.e.

```
x = relative_position[:, 0]
y = relative_position[:, 1]
u = x * width
v = y * height
```